src/js/pouchPersisted.js

```diff
@@ -0,0 +1,174 @@
+/*
+Copyright 2016 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.txt
+*/
+
+/* global fluid, floe, PouchDB */
```

Comment:
It's a bit awkward that this file binds concretely to PouchDB - this interaction would be better mediated through some DataSources which would also make it easier to port to CouchDB etc. This would also encourage us to improve and rethink our CouchDB-aware DataSource support.